source/index.html

```diff
@@ -1,12 +1,170 @@
+        <div class="example__wrapper">
+          <h2 class="example__title">Живой пример</h2>
+          <p class="example__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+          <div class="example__result">
+            <div class="example__result-box">
+              <p class="example__number">5 КГ</p>
+              <p class="example__definition">снижение веса</p>
+            </div>
+            <div class="example__result-box">
+              <p class="example__number">60 ДНЕЙ</p>
+              <p class="example__definition">затрачено времени</p>
+            </div>
+          </div>
+          <p class="example__cost">Затраты на питание: <span>15 000 РУБ.</span></p>
+          <div class="example__cats">
+            <img class="example__img example__img--before" src="img/cat-before.png" width="280" height="232" alt="Толстый кот">
+            <img class="example__img example__img--after" src="img/cat-after.png" width="280" height="232" alt="Худой кот">
+            <div class="select">
+              <button class="select__button" type="button">было</button>
+              <div class="select__range">
+                <button class="select__range-button" type="button"><span class="visually-hidden">изменить положение</span></button>
+              </div>
+              <button class="select__range-button" type="button">стало</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="contacts">
+        <h2 class="visually-hidden">Адрес магазина</h2>
+        <div class="contacts__cooperation">
+          <p class="contacts__cooperation-text">приглашаем к сотрудничеству дилеров!</p>
+          <p class="contacts__address">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</p>
+        </div>
+        <div class="contacts__map">
+          <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1998.6036253003006!2d30.32085871595745!3d59.938719169054345!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4696310fca145cc1%3A0x42b32648d8238007!2z0JHQvtC70YzRiNCw0Y8g0JrQvtC90Y7RiNC10L3QvdCw0Y8g0YPQuy4sIDE5LzgsINCh0LDQvdC60YIt0J_QtdGC0LXRgNCx0YPRgNCzLCDQoNC-0YHRgdC40Y8sIDE5MTE4Ng!5e0!3m2!1sru!2see!4v1637931117542!5m2!1sru!2see" width="320" height="362" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
+          <img class="contacts__map-icon" src="img/map-icon.png" width="57" height="53" alt="Иконка на карте.">
+        </div>
+      </section>
+    </main>
+
+    <footer class="page-footer">
+      <div class="page-footer__wrapper">
+        <img class="logo page-footer__logo" src="img/logo.svg" width="128" height="24" alt="логотип cat energy">
+        <ul class="page-footer__social social">
+          <li class="social__item">
+            <a class="social__link" href="#">
```

Comment:
чтобы на верх страницы не перебрасывало при клике
```suggestion
            <a class="social__link" href="#!">
```